src/Umbraco.Web/HtmlHelperRenderExtensions.cs

```diff
@@ -687,8 +690,10 @@ public static MvcForm BeginUmbracoForm(this HtmlHelper html, string action, stri
                                                IDictionary<string, object> htmlAttributes,
                                                FormMethod method)
         {
-            if (string.IsNullOrEmpty(action)) throw new ArgumentNullOrEmptyException(nameof(action));
-            if (string.IsNullOrEmpty(controllerName)) throw new ArgumentNullOrEmptyException(nameof(controllerName));
+            if (action == null) throw new ArgumentNullException(nameof(action));
+            if (string.IsNullOrEmpty(action)) throw new ArgumentException("Value can't be empty.", nameof(action));
+            if (controllerName == null) throw new ArgumentNullException(nameof(controllerName));
+            if (string.IsNullOrEmpty(controllerName)) throw new ArgumentException("Value can't be empty.", nameof(controllerName));
```

Comment:
Should this be changed to `string.IsNullOrWhiteSpace()` to keep it the same as in other methods?

Reply:
yep